src/sources/socket/tcp.rs

```diff
@@ -22,6 +22,7 @@ pub struct TcpConfig {
     #[serde(default = "default_shutdown_timeout_secs")]
     shutdown_timeout_secs: u64,
     host_key: Option<String>,
+    port_key: Option<String>,
```

Comment:
I wonder if it is more correct for the port key to be a short integer instead (port numbers range from 0-65535)?